CHANGELOG.md

```diff
@@ -9,7 +9,10 @@
 * Added support for tvOS and watchOS in images, fonts and color templates.  
 [Tom Baranes](https://github.com/tbaranes),
   [#145](https://github.com/AliSoftware/SwiftGen/pull/145)
-  
+* Added support for OSX in storyboards.  
```

Comment:
If you have any better formulation / explanation than my update, let me know, or update the CHANGELOG directly :)